lxd/daemon.go

```diff
@@ -482,7 +482,7 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 			}
 
 			// If not a macaroon discharge request, return the error
-			_, ok = err.(*bakery.DischargeRequiredError)
+			_, ok := err.(*bakery.DischargeRequiredError)
```

Comment:
Looks like I missed some macaroon based authentication stuff in #12830. I'll send a follow up PR.

Reply:
Follow up PR here #12860